projects/packages/schema/src/class-schema-context.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Automattic\Jetpack\Schema;
+
+class Schema_Context {
```

Comment:
I'm wondering if we shouldn't drop the `Schema` prefix for these classes. They are already in the `Schema` context.

```php
use Automattic\Jetpack\Schema\Schema_Context
```

```php
use Automattic\Jetpack\Schema\Context
```

I think I prefer the second better, but it's not a strong preference. Up to you.

Reply:
I don't have strong preferences about it either. But, I think it's probably OOS for this PR.